.github/workflows/pipeline-tag-deploy-int.yaml

```diff
@@ -0,0 +1,60 @@
+name: Tag and deploy code to integration account
+on:
+  pull_request:
+    types: [ labeled ]
+
+jobs:
+  metadata:
+    name: "Get Metadata"
+    uses: NHSDigital/uec-dos-management/.github/workflows/metadata.yaml@v0.35
+  generate-tag-name:
+    name: "Generate Tag Name"
+    if: ${{ github.event.label.name == 'Int Ready'  && startswith(github.head_ref, 'task/') }}
+    needs:
+      [
+        metadata,
+      ]
+    uses: NHSDigital/uec-dos-management/.github/workflows/generate-tag-name.yaml@v0.35
+    with:
+      workspace: ${{ needs.metadata.outputs.workspace }}
+      tag_type: int
+      use_commit_hash: "yes"
+      commit_hash: ${{ needs.metadata.outputs.artefact_commit_hash }}
+  tag-repository:
+    name: "Tag Repository"
+    needs:
+      [
+        metadata,
+        generate-tag-name,
+      ]
+    uses: NHSDigital/uec-dos-management/.github/workflows/tag-repository.yaml@v0.35
+    with:
+      tag_to_push: ${{ needs.generate-tag-name.outputs.tag_name }}
+      tag_overwrite: "yes"
+  deploy-to-int:
+    name: Deploy to integration
+    needs: [
+      metadata,
+      generate-tag-name,
+      tag-repository,
+    ]
+    uses: ./.github/workflows/pipeline-deploy-app-int.yaml
+    with:
+      workspace: ${{ needs.metadata.outputs.workspace }}
+      commit_hash: ${{ needs.metadata.outputs.artefact_commit_hash }}
+      # tag: ${{needs.generate-tag-name.outputs.tag_name}}
+      # artefact_bucket_name: ${{ needs.metadata.outputs.artefact_bucket_name }}
+    secrets: inherit
+  slack-notifications:
+    name: Send notification to slack
+    needs: [
+      metadata,
+      generate-tag-name,
+      tag-repository,
+      deploy-to-int
+    ]
+    if: always()
+    uses: NHSDigital/uec-dos-management/.github/workflows/slack-notifications.yaml@v0.35
+    with:
+      env: test
```

Comment:
copy paste error? - env: int

Reply:
Ah I think this may be different - this environment is used to grab secrets one of which is which slack channel to use ; and we are either using the test pipeline channel or we have to create a new  dedicated channel for int and use that (I think we should but maybe a later ticket). I will sound out Tamara - but for now I think we let this ride

Reply:
Ah right sounds good will resolve